src/main/scala/org/downtowndailybread/bethsaida/controller/service/schedule/Delete.scala

```diff
@@ -0,0 +1,29 @@
+package org.downtowndailybread.bethsaida.controller.service.schedule
+
+import akka.http.scaladsl.server.Directives.{JavaUUID, path}
+import org.downtowndailybread.bethsaida.controller.ControllerBase
+import org.downtowndailybread.bethsaida.exception.service.ScheduleNotFoundException
+import org.downtowndailybread.bethsaida.json.JsonSupport
+import org.downtowndailybread.bethsaida.request.ServiceRequest
+import org.downtowndailybread.bethsaida.request.util.DatabaseSource
+import org.downtowndailybread.bethsaida.service.AuthenticationProvider
+
+trait Delete extends ControllerBase {
+  this: JsonSupport with AuthenticationProvider =>
+
+  val schedule_deleteRoute = path(JavaUUID / "schedule" / JavaUUID / "delete") {
+    (serviceId, scheduleId) =>
+      authorizeNotAnonymous {
+        implicit user =>
+          futureComplete({
+            val schedIds =
```

Comment:
I'm really glad you saved those three characters here. I see you use this variable twice, so I guess that's six characters saved.